src/components/CodeAssignmentModal/validation.test.js

```diff
@@ -179,6 +180,7 @@ describe('getErrors', () => {
     expect(result).toEqual({ _error: [BOTH_TEXT_AREA_AND_CSV_ERROR] });
   });
   it('returns a maximum email address error if email address are more than MAX_EMAIL_ADDRESS_ALLOWED', () => {
+    // eslint-disable-next-line no-import-assign
     constants.MAX_EMAIL_ADDRESS_ALLOWED = 3;
```

Comment:
nit: Hmm, feels a bit like a code smell if we have to disable this ESLint rule. Should we instead be mocking the constants in order to change their value vs. mutating the constants directly?